modules/protect.php

```diff
@@ -184,6 +185,22 @@ public function admin_jetpack_manage_notice() {
 	}
 
 	/**
+	 * Filters the login error message to a warning about guessing and becoming locked out.
+	 *
+	 * @param string $error The existing HTML error string.
+	 */
+	public function filter_login_errors( $error ) {
+		global $errors;
+		$err_codes = $errors->get_error_codes();
+
+		if ( in_array( 'invalid_username', $err_codes, true ) || in_array( 'incorrect_password', $err_codes, true ) ) {
+			$error .= __( '<br>If you have forgotten your login information, do not keep guessing as you could be locked out; instead, use the password reset link.' );
```

Comment:
You'll need to add the `jetpack` textdomain to this translation to make sure it can be translated.

I would also recommend that you move the `<br>` out of the translated string to make it easier for translators. Once you've done that, you'll be able to use `esc_html__()` instead of `__()`. There are a few examples about this [here](https://codex.wordpress.org/I18n_for_WordPress_Developers#Translatable_strings).

Thanks!